honeybee_radiance_command/options/rpict.py

```diff
@@ -0,0 +1,963 @@
+        view taken from the standard input. It is not advisable to use this option
+        in combination with the pmblur(1) program, since one takes the place of
+        the other. However, it may improve results with pmblur to use a very small
+        fraction with the −pm option, to avoid the ghosting effect of too few
+        time samples.
+        """
+        return self._pm
+
+    @pm.setter
+    def pm(self, value):
+        self._pm.value = value
+
+    @property
+    def pd(self):
+        """pixel depth-of-field - default: 0.000000
+
+        Set the pixel depth-of-field aperture to a diameter of
+        dia (in world coordinates). This will be used in conjunction with the
+        view focal distance, indicated by the length of the view direction
+        vector given in the −vd option. It is not advisable to use this option
+        in combination with the pdfblur(1) program, since one takes the place
+        of the other. However, it may improve results with pdfblur to use a
+        very small fraction with the −pd option, to avoid the ghosting effect
+        of too few samples.
+        """
+        return self._pd
+
+    @pd.setter
+    def pd(self, value):
+        self._pd.value = value
+
+    @property
+    def ps(self):
+        """pixel sample - default: 4
+
+        Set the pixel sample spacing to the integer size.
+        This specifies the sample spacing (in pixels) for adaptive
+        subdivision on the image plane.
+        """
+        return self._ps
+
+    @ps.setter
+    def ps(self, value):
+        self._ps.value = value
+
+    @property
+    def pt(self):
+        """pixel threshold - default: 0.050000
+
+        Set the pixel sample tolerance to frac. If two samples differ
+        by more than this amount, a third sample is taken between them.
+        """
+        return self._pt
+
+    @pt.setter
+    def pt(self, value):
+        self._pt.value = value
+
+    @property
+    def t(self):
+        """time between reports - default: 0
+
+        Set the time between progress reports to sec.
+        A progress report writes the number of rays traced,
+        the percentage completed, and the CPU usage to the standard error.
+        Reports are given either automatically after the specified interval,
+        or when the process receives a continue (−CONT) signal (see kill(1)).
+        A value of zero turns automatic reporting off.
+        """
+        return self._t
+
+    @t.setter
+    def t(self, value):
+        self._t.value = value
+
+    @property
+    def w(self):
+        """warning messages - default: on
+
+        Boolean switch for warning messages. The default is to print warnings,
+        so the first appearance of this option turns them off.
+        """
+        return self._w
+
+    @w.setter
+    def w(self, value):
+        self._w.value = value
+
+    @property
+    def i(self):
+        """irradiance calculation - default: off
```

Comment:
Do we accept off as a valid input or should all the off/on should be translated to True/False?

Reply:
Maybe I was not clear about the issue that I'm talking about. Does this help?

```suggestion
        """irradiance calculation - default: False
```

We are developing an _interface_ for users to deal with Radiance. The documentation in this file should reflect that. A user should set the `i` to `True` or `False` not `off` and `on`. I'm talking about fixing the docs here not how it will show up in Radiance that is taken care of by boolean type that you linked to.

```python
options.i = True
```